ansible/roles/ocp-workload-optaweb-employee-rostering/tasks/remove_workload.yml

```diff
@@ -5,14 +5,14 @@
 
 - name: define ocp_project
   set_fact:
-    ocp_project: "optaweb-employee-rostering-{{guid}}"
+    ocp_project: "optaweb-employee-rostering-{{ guid }}"
 
 - name: Remove user Project
-  shell: "oc delete project {{ocp_project}}"
+  shell: "oc delete project {{ ocp_project }}"
```

Comment:
Please choose to quote the commands or not and maintain consistency. See line 15 for no quote usage.

Reply:
Haven't done that anywhere - was not told that we are enforcing this. The files are originally mixed up in terms of usage, and some of them are hundreds of lines long. Will not get to doing this before my time on this ends.